tests/integration/session_integration/test.rflx

```diff
@@ -0,0 +1,40 @@
+with Universal;
+
+package Test is
+
+   generic
+      Channel : Channel with Readable, Writable; -- §S-P-C-RW
+   session Session with
+      Initial => Start,
+      Final => Terminated
+   is
+      Message : Universal::Message; -- §S-D-V-T-M, §S-D-V-E-N
+   begin
+      state Start is
+        M : Universal::Message;
```

Comment:
Wrong indentation.

Reply:
fixed in 3c684cf25a88b85b6cf8664ddc5096cd6e4bc2a0